src/memory_manager.rs

```diff
@@ -372,3 +375,42 @@ pub fn get_finalized_object<VM: VMBinding>(mmtk: &'static MMTK<VM>) -> Option<Ob
         .unwrap()
         .get_ready_object()
 }
+
+/// Get the number of workers. MMTk spawns worker threads for the 'threads' defined in the options.
+/// So the number of workers is derived from the threads option.
```

Comment:
It also depends on the `single_worker` feature

Reply:
Sure. I will make the change.